### DISCUSSION TITLE
Feature Request: Add setting: Enable glass on navigation bar

### DISCUSSION THREAD

**Discussion by Umbrella-TEC** [valinet/ExplorerPatcher#2820] [Ideas]
Now the navigation window is white:
![2024-02-14 22_06_57-1022  Работает  - Oracle VM VirtualBox](https://github.com/valinet/ExplorerPatcher/assets/159888040/3776b53a-44a4-437f-8889-678386000b9b)

In Windows 7 the panel has transparency:
![2024-02-14 22_07_28-1022  Работает  - Oracle VM VirtualBox](https://github.com/valinet/ExplorerPatcher/assets/159888040/cb0c3e4b-02df-4367-b64c-b90ea7b819a6)

This feature has been added to OldNewExplorer, but it is not compatible with ExplorerPatcher

**Comment by pyrates999** [comment]
Aero Glass was removed entirely in windows 8.  There's no way to get it back.

**Reply by milestprower92** [reply]
glass8 and dwmblurglass restore aeroglass

**Reply by pyrates999** [reply]
didn't know that :)

**Reply by MollySpacefield** [reply]
also, the "enable glass on navigation bar" means extending the titlebar's background to the navigation bar

**Comment by Amrsatrio** [comment]
This feature is available under Windows 11 but not yet for Windows 10.

**Reply by Umbrella-TEC** [reply]
So I submitted a request for revision.

OldNewExplorer has this feature
